src/game/ingame_menu.c

```diff
@@ -56,8 +56,14 @@ extern u8 course_name_table_jp[];
 extern u8 act_name_table_jp[];
 #endif
 
+#ifdef ENABLE_SPANISH
+extern u8 dialog_table_es[];
+extern u8 course_name_table_es[];
+extern u8 act_name_table_es[];
+#endif
+
 // The language table for the game's dialogs, level names and act names.
-void *languageTable[][3] = {
+void *languageTable[][4] = {
```

Comment:
this isnt right, the dimension is the width of each language entry (which is 3) not the length of the array